dashboard/app/models/pd/enrollment.rb

```diff
@@ -29,6 +29,8 @@
 require 'cdo/code_generation'
 require 'cdo/safe_names'
 
+EMAIL_UNIQUE_ERROR_MESSAGE = 'has already been used to enroll in this workshop. Please contact support@code.org if you need assistance.'
```

Comment:
nit: it feels weird to me to put contact support in this message. Could we instead keep the error message here as the first sentence, and add the "contact support" when showing the error on the frontend?

Reply:
Yeah you're totally right, I was being lazy. Will fix.